packages/components/src/types/forms.ts

```diff
@@ -0,0 +1,29 @@
+import { StringOrTextWithA11y } from './common'
+
+/**
+ * Props that are common to most form elements
+ */
+export type FormElementProps = {
+  /** Optional error text. If present, applies error styling to element */
+  error?: StringOrTextWithA11y
+  /** Header text */
+  header?: StringOrTextWithA11y
+  /** Hint text. Appears below header */
+  hint?: StringOrTextWithA11y
+  /** True to append (*Required) suffix to element */
+  required?: boolean
+  /** True to apply tile styling */
+  tile?: boolean
```

Comment:
Will this be common to any besides the Checkbox(Group) and Radio? Or should it be in the below typing?

Reply:
Moved it below and added it as a local prop type in CheckboxGroup